lib/views/edit/edit_listings.erb

```diff
@@ -0,0 +1,38 @@
+<h1> Edit Your Listings </h1>
+
+
+<% if current_user  %>
+  
+<form id="editform" action="/edit" method='post'>
+  <label for= 'location'>
+    Location: <input type='text' name= 'location' value="<%= @edit_space.location %>"/>
+  </label>
+  <label for= 'name'>
+    Name: <input type='text' name= 'name' value="<%= @edit_space.name %>"/>
+  </label>
+  <label for= 'description'>
+    Description: <input type='text' name= 'description' value="<%= @edit_space.description %>" />
+  </label>
+  <label for= 'details'>
+    Details: <input type='text' name= 'details' value="<%= @edit_space.details %>"/>
+  </label>
+  <label for= 'price'>
+    Price: <input type='text' name= 'price' value="<%= @edit_space.price %>"/>
+  </label>
+  <input type='submit' value='Submit changes'>
+</form>
+
+<form action="/delete" method="post">
+ <!--  <input type='hidden' name='_method' value='delete'/>
+ -->  
+ <input type="submit" value='Delete Listing'>
+</form>
+<% end %>
+
+<% if @current_user == nil %>
```

Comment:
Both `current_user` and `@current_user` are used on the page - one well, the other poorly.